modules/candidate_list/jsx/candidateListIndex.js

```diff
@@ -65,7 +65,7 @@ class CandidateListIndex extends Component {
    * Called by React when the component has been rendered on the page.
    */
   componentDidMount() {
-    fetch('options',
+    fetch('/candidate_list/options',
```

Comment:
If you're going to make this an absolute URL instead of a relative one you need to make sure you include the loris baseurl.